src/Grid/topology.jl

```diff
@@ -107,17 +107,18 @@ function _num_shared_vertices(cell_a::C1, cell_b::C2) where {C1, C2}
 end
 
 function _exclusive_topology_ctor(cells::Vector{C}, vertex_cell_table::Array{Set{Int}}, vertex_table, face_table, edge_table, cell_neighbor_table) where C <: AbstractCell
+    cell_neighbor_ids = Set{Int}()
```

Comment:
Is there an advantage to making this ordered so the order in the `cell_neighbor_table[cell_id]` will not be "random"? Maybe it doesn't matter.

Reply:
Probably this does not matter, because we do not enforce any ordering and locality properties on our grids in general.